examples/webgl_materials_variations_toon.html

```diff
@@ -48,12 +48,21 @@
 				scene = new THREE.Scene();
 				scene.background = new THREE.Color( 0x444488 );
 
+				//
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				container.appendChild( renderer.domElement );
+				renderer.outputEncoding = THREE.sRGBEncoding;
+
 				// Materials
 
 				const cubeWidth = 400;
 				const numberOfSphersPerSide = 5;
 				const sphereRadius = ( cubeWidth / numberOfSphersPerSide ) * 0.8 * 0.5;
 				const stepSize = 1.0 / numberOfSphersPerSide;
+				const format = ( renderer.capabilities.isWebGL2 ) ? THREE.RedFormat : THREE.LuminanceFormat;
```

Comment:
Hmm, is there a way to hide this? 🤔

Reply:
We use this approach at a different location, too:

https://github.com/mrdoob/three.js/blob/38eb8439c6f58188480c08d24f0e56321ff3f758/examples/webaudio_visualizer.html#L116

Hiding this detail in the renderer is problematic since `THREE.RedFormat` behaves differently than `THREE.LuminanceFormat`. If we would internally convert luminance to red, users could be confused because it would be only possible to sample the `r` channel.

IMO, we could leave the code as it is or just use `THREE.RedFormat` in both examples (which would make the WebGL 2 only).